pkgs/top-level/python-packages.nix

```
@@ -410,6 +414,8 @@ in {
 
   imutils = callPackage ../development/python-modules/imutils { };
 
+  inotify_simple = callPackage ../development/python-modules/inotify_simple { };
```

```suggestion
  inotify-simple = callPackage ../development/python-modules/inotify-simple { };
```

Please be sure to change the directory name also.